spikeinterface_gui/main.py

```diff
@@ -51,17 +79,36 @@ def run_mainwindow_cli():
     parser = argparse.ArgumentParser(description='spikeinterface-gui')
     parser.add_argument('analyzer_folder', help='SortingAnalyzer folder path', default=None, nargs='?')
     parser.add_argument('--no-traces', help='Do not show traces', action='store_true', default=False)
-    parser.add_argument('--curation', help='Do not show traces', action='store_true', default=False)
-    
+    parser.add_argument('--curation', help='Enable curation panel', action='store_true', default=False)
+    parser.add_argument('--recording', help='Path to a file or path that can be loaded with load_extractor', default=None)
+    parser.add_argument('--preprocess-recording', choices=['none', 'highpass', 'bandpass'], help='Preprocess the recording', default='none')
```

Comment:
No sure I like this idea.

I we want to have fully preprocessed ecording we should upload somewhere some json files that can be loaded as recording and this file could itself reference a raw zarr file somewhere.

Reply:
maybe we could enable some basic preprocessing in the traceview to be performed on the fly instead?